File: examples/sum.ts
function sumFor(items: number[]): number {
  let total = 0;
  for (let i = 0; i < items.length; i++) {
    total += items[i];
  }
  return total;
}

const sumOf = (numbers: number[]) => {
  let acc = 0;
  for (const n of numbers) {
    acc = acc + n;
  }
  return acc;
};

class Calc {
  add(a: number, b: number): number {
    if (a > b) {
      return a + b;
    } else {
      return b + a;
    }
  }
}

File: examples/sum.py
def sum_while(items):
    total = 0
    i = 0
    while i < len(items):
        total += items[i]
        i = i + 1
    return total


def sum_for(numbers):
    acc = 0
    for n in numbers:
        acc = acc + n
    return acc


class Calc:
    def add(self, a, b):
        if a > b:
            return a + b
        else:
            return b + a
